specs/templates/component-interaction-design.md

```diff
@@ -0,0 +1,23 @@
+# Interaction Design (IxD) [Template]
```

Comment:
Typically interaction designs are better captured in XD/Figma than in a markdown document. Should we update this template to make it clear that it's ok for the bulk of the design to live elsewhere? For example, change "Full description of the proposed interaction design" to "Link to design in XD or Figma and provide any additional context."

Do the other sections even provide value? For example, we don't usually see alternative designs called out in IxD specs. Maybe this whole template could go away and be replaced with a link from the IxD user story to the XD / Figma spec.

Reply:
> It probably doesn't need the "alternative implementations/designs" section.

Removed

>  For example, change "Full description of the proposed interaction design" to "Link to design in XD or Figma and provide any additional context."

Kept both